libraries/standard/http/test/common.h

```diff
@@ -7,4 +7,6 @@
 #include "private/http_client_internal.h"
 #include "private/http_client_parse.h"
 
-#define assert( x )
+static int _assertCount;
+#define assertReset()    do { _assertCount = 0; } while( 0 )
+#define assert( x )      do { _assertCount++; } while( 0 )
```

Comment:
Oh just realized that we only want to increase the _assertCount if x == false

Reply:
Fixed